Instagram/Models/Post.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Post : PFObject <PFSubclassing>
+
+@property (nonatomic, strong) NSString *postID;
+@property (nonatomic, strong) NSString *userID;
+@property (nonatomic, strong) PFUser *author;
+
+@property (nonatomic, strong) NSString *caption;
+@property (nonatomic, strong) PFFileObject *image;
+@property (nonatomic, strong) NSNumber *likeCount;
+@property (nonatomic, strong) NSNumber *commentCount;
+
++ (void) postUserImage: ( UIImage * _Nullable )image withCaption: ( NSString * _Nullable )caption withCompletion: (PFBooleanResultBlock  _Nullable)completion;
```

Comment:
but no newlines here? This is too much for one line. Also the spacing is really inconsistent.

```
+ (void)postUserImage:(UIImage *_Nullable )image
 withCaption:(NSString *_Nullable)caption 
withCompletion:(PFBooleanResultBlock  _Nullable)completion;